gqlgen.yml

```diff
@@ -26,6 +26,10 @@ resolver:
 # Optional: turn on use `gqlgen:"fieldName"` tags in your models
 struct_tag: gqlgen
 
+# Optional: turn off to make struct-type struct fields not use pointers
+# e.g. type Thing struct { FieldA OtherThing } instead of { FieldA *OtherThing }
+struct_fields_always_pointers: false
```

Comment:
What is ther easoning behind this configuration?

Reply:
Since the translation files are defined in the schema and autogenerated, this made each field be a pointer. This isn't ideal behavior for the type, as we desired to show that the fields are require for those types. This turned configuration change created the translation models as expected.